docs/6.x/monitoring.md

```diff
@@ -1,46 +1,67 @@
 
 These components are automatically included into a cluster image built with `tele build` as a
 system dependency (see the [source](https://github.com/gravitational/monitoring-app) on GitHub).
 
-### Heapster
+### Prometheus
 
-Heapster monitors Kubernetes components and reports statistics and information to InfluxDB about nodes
-and pods.
+[Prometheus](https://prometheus.io/docs/introduction/overview/) is an open-source Kubernetes
+native monitoring system and time-series database.
 
-### InfluxDB
+Prometheus uses the following in-cluster services to collect the metrics about the cluster:
 
-InfluxDB is the main data store for current + future monitoring time series data. It provides the
-service `influxdb.monitoring.svc.cluster.local`.
+* [node-exporter](https://github.com/prometheus/node_exporter) collects hardware and OS metrics.
+* [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) collects metrics about
+various Kubernetes objects such as deployments, nodes and pods.
+
+!!! note:
+    Collected metrics are stored for 30 days.
+
+Prometheus exposes the cluster-only service `prometheus-k8s.monitoring.svc.cluster.local:9090`.
 
 ### Grafana
 
-Grafana is the dashboard system that provides visualization information on all the information stored
-in InfluxDB. It is exposed as the service `grafana.monitoring.svc.cluster.local`. Grafana credentials
-are generated during initial installation and placed into a secret `grafana` in `monitoring` namespace.
+[Grafana](https://grafana.com/) is an open-source metrics analytics and visualization suite.
+
+In Gravity clusters Grafana uses Prometheus as a data-source. It is preconfigured with several
+dashboards that provide general information about individual nodes, containers and the overall
+cluster health.
+
+!!! tip:
+    When building a cluster image, it is possible to add your own dashboards in addition to
+    the ones that ship by default. See [Grafana Integration](#grafana-integration) below for
+    details.
+
+Grafana exposes the cluster-only service `grafana.monitoring.svc.cluster.local:3000`.
+
+### Alertmanager
+
+[Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) is a Prometheus component that
+handles alerts sent by Prometheus server and takes care of deduplicating, grouping and routing
+them to the correct receiver such as email recipient.
 
-### Kapacitor
+Alertmanager exposes the cluster-only service `alertmanager-main.monitoring.svc.cluster.local:9093`.
 
-Kapacitor is the alerting system that streams data from InfluxDB and sends alerts as configured by
-the end user. It exposes the service `kapacitor.monitoring.svc.cluster.local`.
+## Grafana Integration
 
-## Grafana integration
+The default Grafana configuration includes two pre-configured dashboards providing machine- and
+pod-level overview of the installed cluster by default. Grafana UI is integrated with Gravity
```

Comment:
of the installed cluster ~~by default~~ (The sentence already hints at the default configuration).